classes/ActionScheduler_Store.php

```diff
@@ -24,6 +44,18 @@
 	 */
 	abstract public function save_action( ActionScheduler_Action $action, DateTime $date = NULL );
 
+	/**
+	 * Update an existing action by ID.
+	 *
+	 * @author Jeremy Pry
+	 *
+	 * @param string $action_id The action ID to update.
+	 * @param array  $fields    The array of field data to update.
```

Comment:
Might be worth saying the field (most likely) needs to be in `$this->action_fields`. Although there is no way to know that for sure, or enforce that in child classes given the generic nature of this method.